app/konnect/dev-portal/access-and-approval/sso.md

```diff
@@ -12,116 +12,9 @@ Keep the following in mind when configuring SSO for Dev Portal:
-    This permission can be revoked later if desired, but one permission is required to create the application.
-
-1. [Configure the Auth0 application settings](https://auth0.com/docs/get-started/applications/application-settings):
-    * **Application Login URI**: `https://cloud.konghq.com/login`
-    * **Allowed Callback URLs**: `https://cloud.konghq.com/login`
-    * **Grant Types**: Authorization Code
-
-1. Create a login [action in Auth0](https://auth0.com/docs/customize/actions/write-your-first-action#create-an-action) and enter the following in the Actions Code Editor:
-    
-    ```js
-    exports.onExecutePostLogin = async (event, api) => {
-    if (event.authorization) {
-        // This transforms the ISO 8601 Timestamp string into the seconds integer representation that is expected for the OIDC standard,
-        // allowing the Konnect SSO validation to accept the format of the `updated_at` property when parsing the token claim.
-        api.idToken.setCustomClaim('updated_at', Math.floor(new Date(event.user.updated_at).getTime()/1000))
-    }
-    };
-    ```
-
-   {:.note}
-   > **Important:** This section is required because the Auth0 API implementation isn't inline with the OIDC standard for the `updated_at` token claim value.
-
-1. Deploy the action by dragging it from the Start to Complete step in the [Login Flow](https://auth0.com/docs/customize/actions/write-your-first-action#deploy-the-action).
-{% endnavtab %}
-{% endnavtabs %}
-
-## Configure SSO in {{site.konnect_short_name}}
-
-From the [{{site.konnect_short_name}} Dev Portal settings](https://cloud.konghq.com/portal/portal-settings), click the **Identity** tab and then click **Configure provider** for **OIDC**. Enter the values from your IdP application.
-
-This table maps the {{site.konnect_short_name}} values to the corresponding IdP values:
-
-| {{site.konnect_short_name}} value      | Azure value | Okta value | Auth0 value |
-| ----------- | ----------- | ----------- | ----------- |
-| Provider URL      | The value stored in the `issuer` variable. | The issuer URI for the authorization server. It should look something like this: `https://example.okta.com/oauth2/default` | Your Auth0 tenant's **Domain** with a leading `https://` and trailing slash `/`, e.g., `https://<your-tenant>.<region>.auth0.com/` |
-| Client ID   | Your Azure application ID.        | The **Client ID** in your Okta application. | Your Auth0 application's **Client ID** value. |
-| Client Secret | Azure client secret.| The **Client Secret** in your Okta application. | Your Auth0 application's **Client Secret** value. |
-
-You can test your configuration by navigating to the Dev Portal and using your IdP credentials to log in. 
\ No newline at end of file
+* [Configure generic SSO for a Konnect Org](/konnect/org-management/sso/)
+* [IdP SSO attribute mapping reference](/konnect/reference/sso)
```

Comment:
Should we add this section for Dev Portal "Configure Okta" page as well: https://docs.konghq.com/konnect/org-management/okta-idp/#add-a-user-to-your-application?

Reply:
I can add a link to that reference page to the Okta page